extension/src/messageHandlers.js

```diff
@@ -5,7 +5,7 @@ const {
 } = require('@lunie/cosmos-keys')
 const {
   default: TransactionManager
-} = require('../lunie/src/signing/transaction-manager')
+} = require('app/src/signing/transaction-manager')
```

Comment:
This means we don't have the submodule there right? 🤩 

Monorepo all the way! This is such a massive improvement! 💃